### ISSUE TITLE
Update pie chart guidelines

### ISSUE THREAD

**Issue by dgkeyes** [rfortherestofus/omni#64]
Per our email, I'll update the pie chart template to look like this:

<img width="335" alt="image" src="https://github.com/OMNI-Institute-QuantBPT/OMNI-R-Resources/assets/13971585/17594359-48d3-47a1-a9ea-e3310ad3ef47">

I'll make sure to make an example with fewer slices and use gray to deemphasize less relevant categories. Anything else you want me to highlight @OskaratOmni?

**Comment by OskaratOmni** [comment]
That looks good David, thanks!

**Comment by dgkeyes** [comment]
Ok, I've updated the [page with pie charts](https://quiet-choux-dfc60e.netlify.app/ggplot/graph_pie_chart.html). I tried something different here: instead of writing everything out, I recorded a video walkthrough (the video is uploading so check back in 15 minutes or so). Personally, I find it much harder to explain complicated things like this in text than on video, and I think it is a really good way to help people understand what's going on. Let me know what you think.
